Code/Legacy/CryCommon/IMovieSystem.h

```diff
@@ -120,18 +120,18 @@ class CAnimParamType
 
```

Comment:
I thought IMovieSystem wouldve been red-coded by now

Reply:
I thought Maestro would have been recorded for now, but it still has it's tentacles in the Editor code.